app/views/partials/download/_download.html.erb

```diff
@@ -1,3 +1,9 @@
-<download text="<%= t('global.button.download') %>">
+<download 
+  <% if local_assigns.has_key? :classes %>
```

Comment:
Is this strictly necessary? If classes is not defined you should get `nil` by default which should then result by an empty string think

Reply:
Yeah I was just wondering if that would cause issues.
But if it's better not to have the class attribute if classes are empty or if HTML doesn't manage this automatically then happy to leave that as is